Modules/@babylonjs/react-native-iosandroid/android/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ cmake_minimum_required(VERSION 3.13.2)
 #   [24, infinite)   ES2 & ES3 & Vulkan
 project(ReactNativeBabylon)
 
+# Disable the WebP plugin option
+set(BABYLON_NATIVE_PLUGIN_NATIVEENGINE_WEBP OFF CACHE BOOL "Disable Babylon Native WebP Support")
```

Comment:
Why do we want to disable this?

Reply:
Adding Webp support is not part of this PR. It is with https://github.com/BabylonJS/BabylonReactNative/pull/665
It's disabled here to not break build.